src/App.tsx

```diff
@@ -15,48 +16,119 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SORT_BY_LENGTH = 'length',
+  SORT_BY_ALPHABET = 'alpha',
+  initialSortField = '',
+}
+
+function sortBy(fieldForSort: SortType, isReversed: boolean): string[] {
```

Comment:
If there are more parameters, you will have to pass the State values ​​in the object. But I think you know, for one value, ok. 
{
    sortField,
    isReversed,
  }